### PR TITLE
moz_kinto_publisher: Publish filter coverage

### LINKED ISSUE #196: moz_kinto_publisher should publish `coverage` information
I've updated the schema for full filters in the `cert-revocations` collection with a `coverage` field.

The coverage information is a list of CT logs and corresponding windows in time. If a certificate from an enrolled issuer has an embedded SCT with a timestamp from any of the given logs, and that timestamp is in the given window, then the filter is guaranteed to cover the certificate.

The `moz_kinto_publisher` job should be updated to populate this field.

### PR DESCRIPTION
**Pull request by jschanck**
Resolves #196

### CODE REVIEW

File: moz_kinto_publisher/main.py [hunk 1]
```diff
@@ -553,10 +559,10 @@ def publish_intermediates(*, args, rw_client):
         try:
```
Comment (mozkeeler) [inline moz_kinto_publisher/main.py:559]: There's a line about 10 up that also uses `record["id"]` to delete an intermediate (thanks for not letting me comment on it, github 🙄) - does that need to change too?

File: moz_kinto_publisher/main.py [hunk 2]
```diff
@@ -646,85 +652,101 @@ def clear_crlite_stashes(*, rw_client, noop):
...
             collection=settings.KINTO_CRLITE_COLLECTION,
-            data=attributes,
-            permissions=perms,
+            fileName=attachment_name,
+            filePath=attachment_path,
+            recordId=recordid,
+        )
+    except KintoException as ke:
+        log.error(
+            f"Failed to upload attachment. Removing stale MLBF record {recordid}: {ke}"
```
Comment (mozkeeler) [inline moz_kinto_publisher/main.py:682]: This is also used for stashes, right? Let's replace `MLBF` with `CRLite` or something more generic.

**Review by mozkeeler — APPROVED** [review]
Seems reasonable. I left a couple of comments.

File: moz_kinto_publisher/main.py (hunk 1 above)
Comment (jschanck) [reply moz_kinto_publisher/main.py:562]: The one above is right, and it's the reason I initially made the type error here. The one above is an error case where we didn't create an `Intermediate` object, so we only have a dictionary. I cleaned it up a bit in the most recent commit.

File: moz_kinto_publisher/main.py (hunk 2 above)
Comment (jschanck) [reply moz_kinto_publisher/main.py]: Done

**Pull request merged**